src/components/OSCALMetadata.js

```diff
@@ -33,8 +37,131 @@ const useStyles = makeStyles((theme) => ({
 // Returns a string with a locality-sensitive representation of this date
 const formatDate = (isoDate) => new Date(isoDate).toLocaleString();
 
+function saveModifiedMetadata(modifiedField, modifiableMetadata, newValue) {
+  /* Since the browser url will be of the form
+   * http://localhost:8080/system-security-plans?url=http://localhost:8080/oscal/v1/ssps/{uuid}
+   * and we just want the URL defined in the url parameter of the browser url, we must find the
+   * location of the ? and increment that value by 5 to get the start of the desired url.
+   */
+  const startOfUrl = window.location.href.indexOf("?") + 5;
+  const url = window.location.href.substring(startOfUrl);
+
+  /* We must specify a method of PATCH so the backend service knows we are just
+   * updating a value.
+   *
+   * We also provide a body so we can specify what needs to be modified.
+   */
+  fetch(url, {
+    method: "PATCH",
+    body: JSON.stringify({
+      metadata: {
+        modifiedField: newValue,
+      },
```

Comment:
Yes, partial updates are expected.  From the back-end story:
> As a developer building apps against the oscal-rest-service I want to be able to update only specific elements of an existing SSP via REST requests so that I can incrementally edit the SSP without transmitting the entire SSP on every call.

Reply:
Also note that the request body needs to maintain the entire JSON model even if only one field is being updated.  If this is for an SSP, `metadata` needs to be within a `system-security-plan` structure.

See the back-end tests for an [example](https://github.com/EasyDynamics/oscal-rest-service/pull/34/files#diff-0fd71ac2e229983b4722ce9d27453b9cbad70e55d56985d1f59ed5f49f115b5eR61).